text/0010-gatsby-docs-localization

```diff
@@ -0,0 +1,104 @@
+* Start Date: 2019-09-11
+* RFC PR:
+* Gatsby Issue:
+
+## Summary
+
+Begin the process of localizing the documentation of Gatsby into non-English languages.
+
+## Motivation
+
+[You Belong Here.](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#you-belong-here)
+
+Gatsby believces that everyone belongs in Open Source, and that includes developers that do not speak English. Having access to translated documentation opens up the ability to create websites to people who don't speak English and don't have the privileges to be able to learn.
+
+## Detailed Design
+
+There are two main approaches we can take for the translations. Whichever path we choose depends on the prorities of the Gatsby team, hence the Request for Comments. The options are: (1) doing translations on GitHub directly or (2) using a translation SaaS platform such as Crowdin. This document will go over both alternatives and the advantages and drawbacks of each option.
+
+## Alternative 1: GitHub-based translations
+
+The first and simplest option is to simply create and host the translations on GitHub itself, along with the rest of the Gatsby documentation. For smaller websites (like [Dan Abramov's blog](https://github.com/gaearon/overreacted.io)) it's enough to put each translation in a markdown file next to the source file and use `gatsby-i18n`.
+
+For larger websites, especially documentation websites that have lots of PRs and require versioning and up-to-date content, this isn't enough. The docs for [Angular](https://github.com/angular/angular-cn/tree/aio), [Vue](https://github.com/vuejs/jp.vuejs.org), and [React](https://github.com/reactjs/es.reactjs.org) store their translations in copies of the repo.
+
+### Advantages
+
+This was the approach that we took for translating the React docs. It appealed to us for a lot of reasons:
+
+* Open Source Contributors tend to be familiar with Git
+* No need to integrate with an external API and software (aside from GitHub)
+* People want to be listed as contributors of the React/Gatsby organization on GitHub, and it's an incentive to contribute.
+* Can use GitHub features such as Code Owners and pull requests as quality assurance
+
+### Drawbacks
+
+The main drawback of this approach is that the Gatsby docs currently live in a monorepo with the rest of the Gatsby source, which means we can't just do a copy of the entire website and use `git merge` like we do with React (and Vue).
+
+This is also an issue if we want translations to live under a path (e.g. `gatsbyjs.org/ja` rather than `ja.gatsbyjs.org`).
```

Comment:
It should be simple to map subdomains to paths and vice versa with any production webserver. but ofc it's better if we don't have to fiddle with it